newdle/client/src/components/creation/ClonePage.js

```diff
@@ -0,0 +1,160 @@
+  );
+  const radioLabel = hasTimeslotsInPast ? (
+    <>
+      <Trans>Keep the timeslots as they are</Trans>
+      <Popup
+        content={t`Some of the current timeslots are in the past`}
+        trigger={<Icon style={{marginLeft: 5}} name="info circle" />}
+      />
+    </>
+  ) : (
+    <Trans>Keep the timeslots as they are</Trans>
+  );
+
+  const canSubmit = option !== 1 || startDate !== '';
+
+  return (
+    <Container text>
+      <div className={styles['advanced-options']} style={{marginTop: 0}}>
+        <div className={styles['headerbar']}>
+          <Header as="h3" className={styles['header']}>
+            <Trans>Cloning options</Trans>
+          </Header>
+        </div>
+        <div className={styles['options']}>
+          <div>
+            <label htmlFor="keepParticipants">
+              <Trans>Keep the list of participants</Trans>
+            </label>
+            <Checkbox
+              className={styles['advanced-checkbox']}
+              id="keepParticipants"
+              toggle
+              checked={keepParticipants}
+              onChange={(_, {checked}) => setKeepParticipants(checked)}
+            />
+          </div>
+          <div>
+            <label htmlFor="keepTimeslots">
+              <Trans>Keep timeslots</Trans>
+            </label>
+            <Checkbox
+              className={styles['advanced-checkbox']}
+              id="keepTimeslots"
+              toggle
+              checked={keepTimeslots}
+              onChange={(_, {checked}) => setKeepTimeslots(checked)}
+            />
+          </div>
+          {keepTimeslots && (
+            <div className={styles['timeslot-options']}>
+              <Radio
+                className={styles['radio']}
+                label={{children: radioLabel}}
+                name="timeslotOptions"
+                checked={option === 0}
+                onChange={() => setOption(0)}
+              />
+              <Radio
+                className={styles['radio']}
+                label={t`Set the timeslots to start at a specific date`}
+                name="timeslotOptions"
+                checked={option === 1}
+                onChange={() => setOption(1)}
+              />
+              {option === 1 && (
+                <div className={styles['date-input']}>
+                  <span>
+                    <Trans>Starting date:</Trans>
+                  </span>
+                  <Input
+                    type="date"
+                    size="small"
+                    value={startDate}
+                    error={startDate === ''}
+                    onChange={(_, data) => setStartDate(data.value)}
+                  />
+                </div>
+              )}
+            </div>
+          )}
+        </div>
+      </div>
+      <div className={styles['create-button']}>
+        <Button color="violet" type="submit" disabled={!canSubmit} onClick={cloneNewdle}>
+          <Trans>Review newdle</Trans> 🍜
```

Comment:
hm... "Review cloned newdle"? and probably worth showing some message on the next page that the new-dle hasn't been saved yet so people do not think they are already done? but we could also see if people realize it on their own first

Reply:
I added an info message to the first step